test/e2e/scenario/agents.scenario.ts

```diff
@@ -270,3 +270,36 @@ describe('user action collection', () => {
     expect(resourceEvents[0].user_action!.id).toBe(userActionEvents[0].user_action.id!)
   })
 })
+
+describe('anchor navigation', () => {
+  it('should not create a new view when it is an Anchor navigation', async () => {
+    await $('#test-anchor')
+
+    await browserExecute(() => {
+      window.location.hash = '#test-anchor'
+    })
```

Comment:
```suggestion
    await $('#test-anchor').click()
```
what about a real behavior?